packages/client/src/v2-events/components/forms/utils.ts

```diff
@@ -147,38 +144,53 @@ export function fieldValueToString({
   value: FieldValue
   intl: IntlShape
   locations: { [locationId: string]: ILocation }
-}) {
+}): string {
   switch (fieldConfig.type) {
+    case FieldType.BULLET_LIST:
+    case FieldType.DIVIDER:
+    case FieldType.PAGE_HEADER:
     case FieldType.DATE:
-      return dateToString(value as DateFieldValue)
     case FieldType.TEXT:
-      return textToString(value as TextFieldValue)
     case FieldType.PARAGRAPH:
-      return paragraphToString(value as ParagraphFieldValue)
+    case FieldType.FILE:
+      // @TODO:
+      return defaultTo(value as string, '')
+
     case FieldType.CHECKBOX:
-      return checkboxToString(value as CheckboxFieldValue)
+      // @TODO: This should be a boolean?
```

Comment:
I think this function used to be the display formatting function, so if checkbox was checked, then in the review view for instance, you saw the value as `Yes` or `No`. Should have been message descriptors tho

Reply:
Ah, sorry the function comment threw me off a bit. But yea, checkbox value should be real boolean and not strings